legacy/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     image: enoriega87/mit-client:latest
     environment:
       - MIT_API=mit
-      - OPENAI_KEY=YOUR_KEY
+      - OPENAI_KEY=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
```

Comment:
@enoriega , you'll want to remove it from here as well...

Reply:
gone